source/DasBlog.Web.UI/DasBlog.Web.csproj

```diff
@@ -13,9 +13,11 @@
     <Platforms>AnyCPU</Platforms>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+    <UserSecretsId>d3583964-0aca-4de4-9521-c74cdf42f990</UserSecretsId>
```

Comment:
What is this UserSercretId?

Reply:
I'm not sure, I believe this is added by AppServices...sorry, I'm struggling to keep my instance separate from the "generic" instance.